.github/workflows/integrations-updater.yml

```diff
@@ -37,23 +37,32 @@ jobs:
           repository: meshery/meshery.io
           token: ${{ secrets.GH_ACCESS_TOKEN }}
           path: meshery.io
+      - name: Checkout cloud repo
+        uses: actions/checkout@master
+        with:
+          repository: layer5io/meshery-cloud
+          token: ${{ secrets.GH_ACCESS_TOKEN }}
+          path: meshery-cloud
       - name: Build mesheryctl
         working-directory: ./meshery/mesheryctl
         run: |
           make;
       - name: Pubish to meshery.io site
         working-directory: ./meshery/mesheryctl
         run: |
-          ./mesheryctl registry publish website ${{ secrets.INTEGRATION_SPREADSHEET_CRED }} 1DZHnzxYWOlJ69Oguz4LkRVTFM79kC2tuvdwizOJmeMw meshery.io/integrations meshery.io/assets/images/integration -o js
+          ./mesheryctl registry publish website ${{ secrets.INTEGRATION_SPREADSHEET_CRED }} 1DZHnzxYWOlJ69Oguz4LkRVTFM79kC2tuvdwizOJmeMw ../meshery.io/integrations ../meshery.io/assets/images/integration -o js
       - name: Pubish to layer5.io site
         working-directory: ./meshery/mesheryctl
         run: |
-          ./mesheryctl registry publish website ${{ secrets.INTEGRATION_SPREADSHEET_CRED }} 1DZHnzxYWOlJ69Oguz4LkRVTFM79kC2tuvdwizOJmeMw layer5/src/collections/integrations layer5/src/collections/integrations -o mdx
+          ./mesheryctl registry publish website ${{ secrets.INTEGRATION_SPREADSHEET_CRED }} 1DZHnzxYWOlJ69Oguz4LkRVTFM79kC2tuvdwizOJmeMw ../layer5/src/collections/integrations ../layer5/src/collections/integrations -o mdx
       - name: Pubish to Meshery docs
         working-directory: ./meshery/mesheryctl
         run: |
           ./mesheryctl registry publish website ${{ secrets.INTEGRATION_SPREADSHEET_CRED }} 1DZHnzxYWOlJ69Oguz4LkRVTFM79kC2tuvdwizOJmeMw docs/pages/integrations docs/assets/img/integrations -o md
-          
+      - name: Pubish to Meshery docs
+        working-directory: ./meshery/mesheryctl
```

Comment:
just checking... we don't need an extra ".", do we? -  `../meshery/mesheryctl`

Reply:
No it is correct, in current Github action context after checking out the other repos we will have file structure something like this, and by default we will be in root, so to run mesheryctl command we will be using `./meshery/mesheryctl` because if we use `../meshery/mesheryctl` it will show no file exist issue

- meshery
  | - [files in meshery directory]
- layer5
  | - [files in layer5 directory]
- meshery.io
  | - [files in meshery.io directory]